Runtime/Exporters/CubemapExporter.cs

```diff
@@ -63,7 +63,7 @@ public override void Export()
 #if UNITY_EDITOR
                 UnityEditor.EditorUtility.DisplayProgressBar("Converting to KTX2", "Converting in progress...", 0.5f);
 #endif
-                ImageUtilies.ConvertToKTX2(Map.exportSettings.OutputDirectory);
+                ImageUtilies.ConvertToKTX2(Path.Combine(Map.projectFolder, ".treasured", Map.projectFolder));
```

Comment:
Should be using Map.exportSettings.OutputDirectory to make it backward compatible